k8s/aat/cluster-00/ccpay/pba-payment-job.yaml

```diff
@@ -19,7 +19,7 @@ spec:
   values:
     job:
       image: "hmctspublic.azurecr.io/payment/jobs:prod-075e6641"
-      schedule: "30 2 * * *"
+      schedule: "*/30 * * * *"
```

Comment:
is this what you updated? didn't we update ccpay-payment-app?

I think you might need more than this PR